flybywiresim/a32nx
# [CDU] Cleanup

## Summary of Changes

- Removed duplicate code
- Removed unused code
- Created NXConstants
- Created NXFunctions
- Fixed some errors
- Simplified some code
- ~~Eventually refactor/rebase the files~~ won't do caused some issues

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub): DerL30N#3751

## Testing instructions

Ensure that everything works as intended or the same way as before.
To give some examples:
- Check that mcdu inputs and messages work
- Check for flight phase changes,
- Check for AP
- Check for AT


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link
